packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -216,9 +216,9 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
     if (isStickyTop || isStickyBottom) {
       let height = 0,
         width = 0;
-      if (this.nonStickyContent) {
-        height = this.nonStickyContent.offsetHeight;
-        width = this.nonStickyContent.scrollWidth;
+      if (this.nonStickyContent && this.nonStickyContent.firstElementChild) {
+        height = this.nonStickyContent.firstElementChild.clientHeight;
```

Comment:
Thanks Harshada for looping me in. Agree with the concerns raised here. This would have serious perf issues, and would add to the perf degradation.
1. offsetHeight would not include the border top/bottom and if the content has border (which we have in DetailsHeader), this would cause issues. @leddie24 any specific reason for using clientHeight here?
2. same concern for the clientWidth. We should use offsetWidth here and also make sure to fix shouldComponentUpdate to use the same. With the current code, shouldComponentUpdate would return true and would cause unncessary renders.

@KevinTCoughlin FYI